spec/support/vcr.rb

```diff
@@ -0,0 +1,8 @@
+require 'vcr'
+
+VCR.configure do |config|
+  config.cassette_library_dir = 'spec/fixtures/vcr_cassettes'
+  config.hook_into :webmock
+
+  config.filter_sensitive_data('<BITRESERVE_AUTH_TOKEN>') { ENV['BITRESERVE_AUTH_TOKEN'] }
```

Comment:
only on the first run, since we need to make an actual request. This is just so that the token does not get printed into the cassettes